pkg/util/cmdline_test.go

```diff
@@ -11,7 +11,7 @@ import (
 
 func Test_parseCmdLineWithPrefix(t *testing.T) {
 	cmdline := `x y harvester.a.b=true "harvester.c=d" harvester.e harvester.f=1 harvester.f=2`
-	m, err := parseCmdLine(cmdline, "harvester")
+	m, err := ParseCmdLine(cmdline, "harvester")
```

Comment:
the `ParseCmdLine` has 3+ places of return error, why the test code simply `t.Fatal`, how does it test the expected error case?

Reply:
I can't comment on that because I haven't worked with this code. The PR is there to convert cmdline parameters to the correct types, not to optimize the existing code. So far, it has served its purpose. I optimize code when it concerns the scope of my task that I am working on with the PR, otherwise I leave it alone because otherwise I would not be able to get anything else done. If I took this approach with every PR, we would have to rewrite Harvester from scratch.